save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/projects/ProjectDemoMenu.kt

```diff
@@ -152,6 +158,10 @@ val projectDemoMenu: FC<ProjectDemoMenuProps> = FC { props ->
         getDemoStatus()
     }
 
+    val (selectedModeCommand, setSelectedModeCommand) = useState<RunCommandPair?>(null)
```

Comment:
I thought you don't like to use `null` in state? 😄

Reply:
Here it is done in order to use `selectedModeCommand` as `WindowOpenness`. If it is null, the modal is hidden

I guess it is ok in this way

> Times they are a-changin'